portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```

Comment:
Delete this TODO now that it's done

Reply:
Will do!

Reply:
I've deleted this in the new commit in the other branch